recipes/espeak-ng/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "espeak-ng" %}
+{% set version = "1.51.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/espeak-ng/{{ name }}/archive/cb62d93fd7b61d8593b9ae432e6e2a78e3711a77.zip
+  sha256: 81f01cdbc928cefde3989e62ee185285c46659503ca0dc1c81e5de2962afd6ad
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ cdt('alsa-lib-devel') }}  # [linux]
+    - cmake
+    - coreutils  # [osx]
+    - autoconf  # [not win]
+    - automake  # [not win]
+    - make  # [not win]
+    - ld_impl_linux-64  # [linux]
+    - libtool  # [not win]
+    - pkg-config  # [not win]
+    - alsa-lib  # [linux]
+    - lld  # [win]
+    - m2-autoconf  # [win]
+    - m2-automake-wrapper  # [win]
+    - m2-base  # [win]
+    - m2-findutils  # [win]
+    - m2-libtool  # [win]
+    - m2-make  # [win]
+    - m2-pkg-config  # [win]
+    - getopt-win32  # [win]
+    - pcaudiolib
+  host:
+    - getopt-win32  # [win]
+    - pcaudiolib
+  run:
+    - getopt-win32  # [win]
+    - pcaudiolib
+
+test:
+  commands:
+    - test -f $PREFIX/bin/espeak-ng  # [not win]
```

```suggestion
    - espeak-ng --help
```
```suggestion
    - espeak-ng --version
```
Can you test whether the binary is executable instead?

---

These seem to be working. We could go with any of the two.